src/components/image-wrapper/image-wrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import CozyClient from 'cozy-client'
+import React, { ReactChild } from 'react'
+import { useState } from 'react'
+import { useEffect } from 'react'
+import ServiceClient from 'lib/collab/stack-client'
+
+export const ImageWrapper: React.FunctionComponent<{
+  children: ReactChild
+  url: string
+  setImage: (image: string) => void
+  setImageError: (error: any) => void
+}> = ({ children, url, setImage, setImageError }) => {
+  const [hasUrl, setUrl] = useState(false)
```

Comment:
This state seems to be used only to limit the number of calls to useEffect().

Would it be the same to remove this state and call useEffect() with an empty array (so it is called once)?

Reply:
Updated